fx-src/index.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
```

Comment:
@rnewman Apache2 is fine: https://www.mozilla.org/en-US/MPL/2.0/FAQ/ - see Q13

MPL2 is my default, but kinto was Apache2 before I showed up..